lib/reporters/mini.js

```diff
@@ -11,9 +11,9 @@ var repeating = require('repeating');
 var colors = require('../colors');
 
 chalk.enabled = true;
-Object.keys(colors).forEach(function (key) {
+for (let key of Object.keys(colors)) {
 	colors[key].enabled = true;
-});
+}
```

Comment:
> but it caused this test to fail because chalk and colors did not seem to work as intended, https://github.com/avajs/ava/blob/master/test/cli.js#L73

I think that's as expected as we used to force Chalk in child processes and we no longer do. I guess we can just remove the `colors.error()` call from that test.